packages/babel-plugin-proposal-pipeline-operator/test/fixtures/fsharp/await/input.js

```diff
@@ -0,0 +1,6 @@
+async function myFunction(n) {
+  return n
+    |> Math.abs
+    |> Promise.resolve
+    |> await;
+}
```

Comment:
We should probs add an assertion / expectation to this test.

Reply:
There is an expectation in `packages/babel-plugin-proposal-pipeline-operator/test/fixtures/fsharp/await/exec.js`. Do you think we need another one?

Reply:
I'm honestly not sure what the diff is between `input.js` & `exec.js`. It looks like most of the tests have both and... they're the same? Dunno. @nicolo-ribuado could you shed some light here?

Reply:
`exec` tests are executed, `input` is compared with output. In `exec` tests you should always use Jest's `expect()`.

Reply:
What's the reason for both?

Reply:
`exec` guarantees runtime behaviour. `input/output` can be used to check for optimizations that do not show up at runtime, for example.

I guess `input` could be copied from `exec` at test run. But I believe right now if `input` doesn't exist `output` won't get checked.

Reply:
Fair enough. The split seems a little odd but that's outside the scope of this PR.